pkg/k8s/apis/cilium.io/v1alpha1/tracing_policy_types.go

```diff
@@ -90,6 +90,10 @@ type TracingPolicySpec struct {
 	// PodSelector selects pods that this policy applies to
 	PodSelector *slimv1.LabelSelector `json:"podSelector,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// ContainerSelector selects containers that this policy applies to
+	ContainerSelector *slimv1.LabelSelector `json:"containerSelector,omitempty"`
```

Comment:
I think it's worth documenting how the label selector is used in the container case.
As far as I understand based on the code, we use the `name` as a label `key` to match the container's name.
Here seems like a good place to document how this works.

Reply:
You are right, it's worth documenting it inside the actual CRD.
I added some documentation. Hopefully, it looks clear :smiley: